components/centraldashboard/app/api.ts

```diff
@@ -27,6 +32,16 @@ export class Api {
     return this.platformInfo;
   }
 
+  private getProfileAndAuth(req: express.Request): [DefaultApi, AuthObject] {
+    const controller = this._controller || (
+      this._controller = new DefaultApi(`${req.get('host')}/kfam`)
```

Comment:
Resolving the user-facing hostname and cluster-level path would work if the request were coming from the browser, but within the cluster it seems like we would unnecessarily be routing the request back out to the public internet only to have it arrive back at the external LB.

I think we can determine the internal endpoint from the K8s service name from within the cluster so the traffic can be handled internally.

Reply:
Resolved, with an in-cluster host.